rs/protobuf/src/registry/node.rs

```diff
@@ -1,3 +1,21 @@
+use crate::registry::node::v1::NodeType;
+
 #[allow(clippy::all)]
 #[path = "../gen/registry/registry.node.v1.rs"]
 pub mod v1;
+
+impl From<NodeType> for String {
+    fn from(value: NodeType) -> Self {
```

Comment:
FYI, `NodeType` already comes with an `as_str_name()` method (and a `from_str_name()` one), although the output is all caps and more verbose.

Reply:
I'm aware, but I couldn't find an option in prost to either remove that, or override the values.